introduction.md

```diff
@@ -1,29 +1,8 @@
 Hello guys, new file!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?